src/centreon/plugins/snmp.pm

```diff
@@ -1023,6 +1144,10 @@ Auto reduce SNMP request size in case of SNMP errors (By default, the divisor is
 
 Use snmp getnext function (even in snmp v2c and v3).
 
+=item B<--snmp-cache-file>
+
+Use SNMP cache file.
```

Comment:
Maybe here we can say how to do it ? Or just something like "use this mode to do that" ?